AppDashboard/templates/apps/database.html

```diff
@@ -0,0 +1,88 @@
+<div class="row-fluid">
+    <div class="box gradient">
+        <div class="title">
+            <h4>
+                <span id="icon" class="icon16 icomoon-icon-search-3"></span>
+                <span id="title">Database & Application Information</span>
+            </h4>
+        </div>
+        <div class="content">
+            <div id="database_info">
+                <h4><span class="icon12 icomoon-icon-database"></span>Current Database:</h4>
+                <!-- DB Hook -->
+                <ul class="unstyled">
+                    <li id="db-img"><span class="icon16 icomoon-icon-arrow-right-2"></span><strong>Powered by
+                        {% if dbinfo['table'] == 'cassandra' %}
+                        <img id="Cassandra" src="/images/cassandra.jpg" alt="Cassandra"/>
+                        {% elif dbinfo['table'] == 'hbase' %}
+                        <img id="Hbase" src="/images/hbase.png" alt="Hbase"/>
+                        {% elif dbinfo['table'] == 'hypertable' %}
+                        <img id="Hypertable" src="/images/hypertable.jpg" alt="Hypertable">
```

Comment:
I know you didn't add these, but the references and assets for Hbase and Hypertable can be removed.